roles/iam-user/tasks/create_users.yml

```diff
@@ -0,0 +1,26 @@
+
+---
+- name: Create IAM users
+  iam:
+    state: present
+    iam_type: user
+    name: "{{ item.name }}"
+    groups: "{{ item.groups }}"
```

Comment:
limit group membership to a single group and add the limitation to the readme. the one to many mapping is not a requirement